### DISCUSSION TITLE
Aproval of Student package

### DISCUSSION THREAD

**Discussion by DCornOnline** [community/community#22672] [New to GitHub]
<p>I was denied access to the student pack because my enrollment image (what I used for proof of student) I had blacked out personal information on the page is that why? I have just submitted another request with it undoctored. Is that going to work? I also have my transcript i got from my college account, but it says it is unofficial due to this being my first semester. which should i use,</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>We don’t have visibility on your application here, but it does sound likely that if incomplete documentation was submitted that they would decline the application.</p>
<p>Transcripts (even ones that say unofficial) should also be acceptable documentation. If you have specific questions about your application, you’ll want to contact our support team directly:</p>
<p><a href="https://support.github.com/contact" class="onebox" target="_blank">https://support.github.com/contact</a></p>
